bindings/python/migration_guide.md

```diff
@@ -0,0 +1,114 @@
+# Migration from `v0.8.0` to `v.0.11.0`
+
+To demonstrate the API changes from `v0.8.0` to `v.0.11.0`, we will use the keras example
+which is available in both versions. For reasons of clarity, some parts of the code have
+been removed.
+
+## [`v0.8.0`](https://github.com/xaynetwork/xaynet/blob/v0.8.0/python/sdk/xain_sdk/participant.py#L24)
+
+```bash
+pip install xain-sdk
+```
+
+```python
+from xain_sdk import ParticipantABC, configure_logging, run_participant
+
+class Participant(ParticipantABC):
+    def train_round(
+        self, training_input: Optional[np.ndarray]
+    ) -> Tuple[np.ndarray, int]:
+        if training_input is None:
+            self.regressor = Regressor(len(self.trainset_x.columns))
+            return (self.regressor.get_weights(), 0)
+
+        return (self.regressor.get_weights(), self.number_of_samples)
+
+    def deserialize_training_input(self, data: bytes) -> Optional[np.ndarray]:
+        if not data:
+            return None
+
+        reader = BytesIO(data)
+        return np.load(reader, allow_pickle=False)
+
+    def serialize_training_result(
+        self, training_result: Tuple[np.ndarray, int]
+    ) -> bytes:
+        (weights, number_of_samples) = training_result
+
+        writer = BytesIO()
+        writer.write(number_of_samples.to_bytes(4, byteorder="big"))
+        np.save(writer, weights, allow_pickle=False)
+        return writer.getbuffer()[:]
+
+def main() -> None:
+    participant = Participant(args.data_directory)
+
+    run_participant(
+        participant, args.coordinator_url, heartbeat_period=args.heartbeat_period
+    )
+```
+
+## `v0.11.0`
```

Comment:
maybe a link similar to the v0.8 header

```suggestion
## [`v0.11.0`](https://github.com/xaynetwork/xaynet/blob/v0.10.0/bindings/python/xaynet_sdk/participant.py)
```

the link will be broken until we release v0.11, but that shouldn't matter much.